### ISSUE TITLE
HTML5 multiple tgroups render in a single table

### ISSUE THREAD

**Issue by DavidMulder0** [dita-ot/dita-ot#3557]
## Expected Behavior
Each `<tgroup>` has its own column specification. The first tgroup having 3 columns and the second only 1 shouldn't affect the presentation of the second tgroup.

## Actual Behavior
An HTML `<table>` gets generated with two `<tbody>`s, but the column specification of the first `<tgroup>` affects the second `<tgroup>`. In other words, this happens:

![image](https://user-images.githubusercontent.com/20395438/89717383-777a4b00-d9b6-11ea-8eb1-0dabc86076b0.png)

![image](https://user-images.githubusercontent.com/20395438/89717407-b8725f80-d9b6-11ea-9d41-d2ed3642a953.png)

## Possible Solution
A single DITA `<table>` with two `<tgroup>`s should in HTML render two `<table>`s.

## Environment

* DITA-OT version:
  DITA-OT version 3.4.1
* Operating system and version:
  Windows
* How did you run DITA-OT?
  `dita` command
* Transformation type:
  HTML5

**Comment by jelovirt** [comment]
IMO this would be the correct solution, but unfortunately for backwards compatibility this difficult to change.

**Comment by DavidMulder0** [comment]
Yeah, I totally understand that backwards compatibility is always a concern, although I wonder whether it might not just be that nobody used multiple `tgroup`s so far. A `<table>` with a single `<tgroup>` could still render exactly the same as now. The only 'odd' thing would be that the second table wouldn't have a `<caption>` element.
